prac_06/guitars.py

```diff
@@ -0,0 +1,15 @@
+from prac_06.guitar import Guitar
+
```

Comment:
I wrapped my code in a main() function for this program (and for all my client code programs) - not sure if LW expects this?

Reply:
I am not sure either, but it is probably best practice to do so so thanks for pointing it out.